### ISSUE TITLE
Plugin very slow when using with other preset 

### ISSUE THREAD

**Issue by mathieumure** [sveltejs/eslint-plugin-svelte3#92]
Hi,

I notice performances issues when using this plugin with other eslint config preset such as `eslint-config-airbnb-base`.

For instance, I got 7 svelte files and it tooks 11seconds to lint with 'airbnb-base' active and only 1s when I removed it.

Here is my .eslintrc file :

```js
module.exports = {
  env: {
    browser: true,
    es2021: true,
  },
  extends: ['airbnb-base', 'prettier'],
  ignorePatterns: ['public/build'],
  parser: '@typescript-eslint/parser',
  parserOptions: {
    ecmaVersion: 2019,
    sourceType: 'module',
  },
  plugins: ['svelte3', '@typescript-eslint'],
  overrides: [
    {
      files: ['*.svelte'],
      processor: 'svelte3/svelte3',
      rules: {
        'import/first': ['off'],
        'import/no-duplicates': ['off'],
        'import/no-mutable-exports': ['off'],
      },
    },
    {
      files: ['.eslintrc.js', '.prettierrc.js'],
      rules: {
        'global-require': ['off'],
        'import/no-extraneous-dependencies': ['off'],
      },
    },
  ],
  settings: {
    'svelte3/typescript': require('typescript'),
  },
  rules: {},
};

```

Can you take a look ? Thanks  🙏

**Comment by clintwood** [comment]
Hi, I began having the same issue the moment I switched to using `svelte-preprocess` and `type="ts"` for svelte files.  I am also using `airbnb-base` but not `prettier`.  From what I can see this is because `airbnb-base` includes `import/*` rules which are very useful.  Prior to using a preprocess config linting would take about 2-3 seconds.  Now it takes 3 minutes.  Here are the timings for my lint with preprocessing (which only includes TS preprocessing)
![image](https://user-images.githubusercontent.com/2149388/112510588-76e9c580-8d89-11eb-984e-c83191bd6ab7.png)
I'm not convinced your PR #93 will make much difference.  I think to resolve this there needs to be some caching strategy for `import/*` rules because it looks like it is literally compiling each import for each svelte file for each rule check.
